source/checklist.rst

```diff
@@ -0,0 +1,18 @@
+Onboarding Checklist
+====================
+
+When you join the lab, these are the steps for onboarding you will undergo:
+
+ 1. Go over this document with Matt
+    a. Code of Conduct
+    b. Communication modes
+    c. Expectations around openness
+ 2. Get added to the DXL groups
+    a. Mailinglist
+    b. Github (and Bitbucket)
+    c. Slack
+    d. Box folder for `git-annex`
+ 3. Add yourself to the `data-exp-lab.github.io` webpage (and a blog post about
```

Comment:
You use "blog post" here but then on line 9 of `communication.rst` you use "news items". I would maybe change this to "blog post" so it's clear they're not expected to write something long for their welcome-to-the-lab-type post.